### ISSUE TITLE
Routing issues for LAN packets with split-tunneling

### ISSUE THREAD

**Issue by TobyGiacometti** [mullvad/mullvadvpn-app#2101]
## Issue report

Operating system: Debian GNU/Linux 10 (Buster)

App version: 2020.05

### Issue description

When a process is started with `mullvad-exclude`, LAN packets from the process are not being routed correctly. LAN packets should not be marked by the firewall. This works (4th line was added):

```
table ip mullvadmangle4 {
	chain mangle {
		type route hook output priority -150; policy accept;
		ip daddr { 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16, 169.254.0.0/16 } accept
		cgroup 5087041 ct mark set 0x00000f41 mark set 0x00000f41
	}
}
```

Same for the *mullvadmangle6* table.

**Comment by faern** [comment]
Thanks for the report. We will look into it

**Comment by dlon** [comment]
This was fixed in 2020.6-beta3.
